docs/content/docs/plugins/BuildChecks.md

```diff
@@ -0,0 +1,264 @@
+---
+title: Build checks plugin
+type: docs
+---
+
+# Build checks plugin
+
+This plugin verifies [common build problems]({{< relref "#checks">}}) with environment and project configuration.
+
+## Getting started
+
+Apply the plugin in the root `build.gradle` file:
+
+```groovy
+plugins {
+    id("com.avito.android.buildchecks")
+}
+
+buildChecks {
+    enableByDefault = true
+}
+```
+
+In the `settings.gradle`:
```

Comment:
i'd like to move this section on separate page, and link every plugin how-to's first section there

we already have this info btw https://avito-tech.github.io/avito-android/docs/infrastructure/

Reply:
I know. I've copied this info from infrastructure page intentionally.
As a user, I'd prefer to see all information in a tutorial, without walking to different pages.

We can include this block to have a single source of truth. Like we did for `{{avito}}` shortcode. 
What about such approach? I can do it but in separate PR.

Reply:
sounds great